units/host_monitoring_idle_termination.go

```diff
@@ -165,11 +159,11 @@ func (j *idleHostJob) checkAndTerminateHost(ctx context.Context, h *host.Host, d
 	idleTime := h.IdleTime()
 	communicationTime := h.GetElapsedCommunicationTime()
 
-	idleThreshold := idleTimeCutoff
+	idleThreshold := d.HostAllocatorSettings.AcceptableHostIdleTime
 	if h.RunningTaskGroup != "" {
-		idleThreshold = idleTaskGroupHostCutoff
+		idleThreshold = idleThreshold * 2
 	} else if hostHasOutdatedAMI(*h, d) {
-		idleThreshold = outdatedIdleTimeCutoff
+		idleThreshold = idleTime / 2
```

Comment:
This change does increase the idleThreshhold from 1 minute to 2 minutes, right? I think one minute is a standard enough duration that we don't need it to be based on the host allocator setting.